CHANGELOG.md

```diff
@@ -15,6 +15,8 @@ Versioning](https://semver.org/spec/v2.0.0.html) for `libnopegl`.
 - `ngl_get_viewport()` to get the viewport currently in use by the rendering
   context
 - `FontFace.index` parameter to select a different face in the font file
+- `DrawTexture.texture` now accepts transform nodes before the texture node to
+  serve as a reframing mechanism (in a texture centered `[-1,1]` space)
```

Comment:
(in a texture centered `[-1,1]` space) -> (the transforms are applied to the texture coordinates in the `[-1,1]` space with the origin at the top left)

Reply:
bottom-left? 🤔 It's the "screen space" so it's centered

Reply:
well, we need to be clear about where (-1,-1) is, as the uv coordinates (0,0) is top left for example.

Reply:
but the origin is not at `[-1,1]`, it's at `[0,0]`, in the center

Reply:
Changed to: `the transforms are applied to the texture coordinates in a centered `[-1,1]` space with `(-1,-1)` in the bottom left`